### DISCUSSION TITLE
c/

### DISCUSSION THREAD

**Discussion by giscus** [OweQian/oweqian.github.io#13] [General]
# c/

朋友花了 700 多块送的 STM32 开发板，钱不能白花，我要先把 C 语言学会，加油吧！

http://oweqian.xyz/c/

**Comment by GOWxx** [comment]
学习嵌入式开发，从 `STEAM32` 修正成 `STM32` 开始 :smile:

**Reply by OweQian** [reply]
😄嘿嘿，有进步了哈。
